omnia.yml

```diff
@@ -17,6 +17,27 @@
   ansible.builtin.import_playbook: utils/check_package_lock.yml
```

Comment:
lets keep utility as first task for all playbook. package lock can be moved to down